PaTTAN Room Displays/PaTTAN Room Displays/DualRoomPage.xaml.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
```

Comment:
Are all these **using** directives needed?

Reply:
According to the IDE none of them are.